cmd/cmd.go

```diff
@@ -78,7 +78,7 @@ func Main(embeddedWebsite fs.FS) {
 	// start the application server
 	// TODO: customized listen address?
 	// TODO: Use a host like tlz.localhost to serve HTTP/2 over HTTPS... just need to automate the CA and cert... - or maybe a public domain like timelinize.app or timelinize.run or something
-	startedServer, err := app.Serve("")
+	startedServer, err := app.Serve(os.Getenv("TIMELINIZE_ADMIN_ADDR"))
```

Comment:
I don't want to rely on env variables long-term, but this is a workable temporary solution until we figure out how configuration will look.

Reply:
Makes total sense.